src/unisparks/player.hpp

```diff
@@ -200,6 +200,8 @@ class Player {
 
   Network* network() const { return network_; };
 
+  bool powerlimited;
```

Comment:
Unless I'm missing something, I don't think this variable needs to be on this class, it can just be a local stack variable where it's used?